scripts/build-win.py

```diff
@@ -39,6 +38,8 @@
 def gngen(arch, ssl_root, msdk_root, quic_root, scheme, tests):
     gn_args = list(GN_ARGS)
     gn_args.append('target_cpu="%s"' % arch)
+    if arch != 'arm64':
```

Comment:
if arch!='arm64' and scheme=='release', there will be two `is_debug=false`.

Reply:
I don't see the logic for this, the conditional for `if arch != 'arm64':` is to set `is_clang=false`, the next conditional will only set `is_debug=false`

Reply:
You're right. Sorry for my incorrect comment above.